tests/test_execute_command.py

```diff
@@ -0,0 +1,58 @@
+from unittesting import DeferrableTestCase
+import sublime
+from os.path import dirname
+from LSP.plugin.core.types import ClientConfig, ClientStates, LanguageConfig
+from LSP.plugin.core.test_session import MockClient
+from LSP.plugin.core.sessions import Session
+from LSP.plugin.core.registry import windows  # , session_for_view
+from LSP.plugin.core.settings import client_configs
+
+test_file_path = __file__
```

Comment:
@tomv564 I got the tests to work but only with a python file using the `testfile.txt` with scope `text.plain` didn't work for the client.
See: https://github.com/tomv564/LSP/pull/474#discussion_r241889838
While debugging the issue I discovered that the file content was somehow 0 (using view.size()) for sublime, I don't why, but opening a python file solved the issue and selector score was > 0